crates/sui-indexer/src/models_v2/epoch.rs

```diff
@@ -33,7 +33,8 @@ pub struct StoredEpochInfo {
     pub epoch_commitments: Option<Vec<u8>>,
 }
 
-#[derive(Queryable)]
+#[derive(Queryable, Clone, Debug, Selectable)]
```

Comment:
We may not need Clone and Debug not sure

Reply:
As long as we have them in our graphql types, we'd need them. I will try without them everywhere and see how we do.